esp/src/src/KeyValStore.ts

```diff
@@ -56,6 +56,34 @@ store.get("browser-stats").then(statsStr => {
     }
 });
 
+export function fetchStats() {
```

Comment:
Is this considered core functionality of our KeyVal logic that we are adding more code in the implementation file vs. in its own file?